Core1Platform/UserInterface/02_Dashboard.md

```diff
@@ -0,0 +1,211 @@
+    - *Statistics by Channel*   
+            Displays the statistics of each connection.
+    - *Connector Summary*   
+            Displays a summary of each connection.
+    - *Statistics over Time*   
+            Displays the statistics of each connection over time.
+    - *Connection Status*   
+            Displays the status of each connection. You can display all active or inactive connections.-->
+            
+    **Actindo DataHub**   
+        For detailed information, see [Dashlet picker](../../DataHub/UserInterface/01a_Dashboards.md#dashlet-picker "Dashlet picker") in the *Actindo Channels* documentation.
+
+    <!---- *Attribute Count*   
+            Displays the number of attributes in the *DataHub* module.
+    - *Language Count*   
+            Displays the number of languages in the *DataHub* module.
+    -	*Set Count*   
+            Displays the number of attribute sets in the *DataHub* module.
+    -	*Entity Type Count*   
+            Displays the number of entity types in the *DataHub* module.
+    -	*Group Count*   
+            Displays the number of groups in the *DataHub* module.
+    -	*Channel Count*   
+            Displays the number of channels in the *DataHub* module.
+    -	*Entity Values by Entity Type*   
+            Displays a ring chart with the distribution of entity values by entity type in the *DataHub* module.
+    -	*Mapping Errors*   
+            Displays a chart with the number of mapping errors in the *DataHub* module.
+    -	*New and Changed Values*   
+            Displays a line chart with the number of new and changed values within a certain period of time in the *DataHub* module.
+    -	*Modified Values by Set for Today*   
+            Displays a chart with the number of today's modified values within an attribute set in the *DataHub* module.
+    -	*Current Mapping Statistics*   
+            Displays a bar chart with the average and current number of mappings. These numbers are displayed for the waiting, the successful and all mappings in the *DataHub* module.
+    -	*ETL Mapping History*   
+            Displays a chart with the history of mapping number for the waiting, the successful and all mappings in the *DataHub* module.
+    -	*DataHub Flow*   
+            Displays a flow chart with connection of mappings between the different entities. By enabling the *Show Disconnected Nodes* toggle, also those entities without any mapping connection are displayed in the flow chart.   -->
+
+    **Fakturierung (Order Management System (OMS))** <!---Hallo Julian, diese hier habe ich mir zusammengereimt--> 
+    - *Orders*  
+            Displays the number of orders for the specified time interval. In addition, it shows the ratio in percent compared to the previous month.
```

Comment:
This dashlet displays... 
(konsistent mit der verlinkten Doku in anderen Modulen)

Reply:
Schlimm genug, dass da 30 mal displays steht..... die verlinkte Doku kommt sowieso raus